hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/AdminService.java

```diff
@@ -339,8 +360,29 @@ public synchronized void transitionToActive(
   }
 
   @Override
-  public synchronized void transitionToStandby(
+  public void transitionToStandby(
       HAServiceProtocol.StateChangeRequestInfo reqInfo) throws IOException {
+    if(rm.rmContext.isHAEnabled()){
```

Comment:
We need to pay attention to the  indentation  of the code

Reply:
Thanks for your review.  I have already formatted it.